x11-misc/slim/slim-1.4.0.ebuild

```diff
@@ -0,0 +1,128 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake pam systemd
+
+if [[ ${PV} == "9999" ]] ; then
```

Comment:
There is no `slim-9999.ebuild`, did you add one and forgot to push it?

Reply:
I use one locally for testing - I'm not sure whether it would be desirable to add it. Which way would you prefer I fix this?

Reply:
Add it and keep it in sync with the last version, no point in keeping it private since you already use it :)